index.html

```diff
@@ -0,0 +1,318 @@
+                       </li>
+                   </ul>
+               </div>
+               <br>
+               <button type="button"><img src="images/Getmyresume.png" alt=""></button>
+               </div>
+               <div class="myskills">
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Languages.png" alt=""></li>
+                   <li class="greaterthan"><a href="">&#86;</a></li>
+                </ul>
+               </div>
+               <div class="frameworks">
+                   <ul>
+                       <li><img src="images/Frame12.png" alt=""></li>
+                       <li><img src="images/Frame13.png" alt=""></li>
+                       <li><img src="images/Frame14.png" alt=""></li>
+                   </ul>
+               </div>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Frameworks1.png" alt=""></li>
+                   <li class="lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Skills1.png" alt=""></li>
+                   <li class="skills-text lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+            </div>
+           </div>       
+           </section>
+
+           <section class="Contact-part">
+            <div class="headline1">
+              
+                 <h2 class="mainheader">
+                     Contact Me</h2>
+                 <p class="details-2">
+                    If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it.
+                 </p>
+               
+               <div class="form-input">
+                <form action="https://formspree.io/f/mayvkezq" method="post">
+                    <div class="container">
+                      <input type="text" placeholder="Name" name="Name" id="name" maxlength="30" required>                
+
+                      <input type="email" placeholder="Email" name="email" id="email" required>
+                  
+                      <textarea class="textbox" placeholder="Write your message here" name="comments" id="comments" maxlength="500" required></textarea>                                      
+                      
+                    </div>           
+                    <button type="submit" class="registerbtn">Get It Touch</button>    
+                    
+                  </form>
+               </div>   
+           </div>    
+           
+           </section>  
+        </main>
+</body>
```

Comment:
- [ ]  You've done a great Job to make the changes suggested in the previous review but you only improved your desktop design only.  Consider your styling outside your media queries to fix you mobile design referring to the one from the [template](https://rawcdn.githack.com/JOdiemo/contact-desktop-version/6da11831d196bb1e8b3cd8f76d3f1f5968f0b346/index.html)

![image](https://user-images.githubusercontent.com/43877689/173176764-1ccb490c-aba7-4bbc-b788-c9093a73a82c.png)